validphys2/src/validphys/plotutils.py

```diff
@@ -26,6 +26,8 @@
 
 log = logging.getLogger(__name__)
 
+DEFAULT_FIGSIZE = (6.4, 4.8)
```

Comment:
For me, it is sufficiently clear as it is (the variable name is clear enough), but just to inform you: Sphinx has also a syntax for documenting these variables, i.e. just place a docstring right after:

```py
MY_VAR = ...
"""My docstring.

Bla bla bla...

"""
```

If you really want to add docs, I'd suggest to just limit to a single line:
```py
MY_VAR = ...
"""My docstring."""
```

Reply:
Btw: of course any string literal is a docstring, the triple quotes are not required.

Reply:
Please remove this. This is set in the matplotlib style and should be there.